onnx/backend/test/case/node/reverse.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Reverse(Base):
+
+    @staticmethod
+    def export_default():  # type: () -> None
+        input = np.arange(6.0).reshape(2, 3)
+
+        node = onnx.helper.make_node(
+            'Reverse',
+            inputs=['input'],
+            outputs=['output']
+        )
+
+        output = np.flip(input)
+
+        expect(node, inputs=[input], outputs=[output],
+               name='test_reverse_default')
+
+    @staticmethod
+    def export_with_axes():  # type: () -> None
+        input = np.arange(6.0).reshape(2, 3)
+
+        node = onnx.helper.make_node(
+            'Reverse',
+            inputs=['input'],
+            outputs=['output'],
+            axes=[0]
+        )
+
+        output = np.flip(input)
```

Comment:
Is this correct? Shouldn't the axes=0 be passed to np.flip?

Reply:
the default is to reverse on all axes.